tests/x509/test_x509.py

```diff
@@ -912,6 +912,44 @@ def test_tbs_certificate_bytes(self, backend):
             cert.signature_hash_algorithm,
         )
 
+    @pytest.mark.supported(
+        only_if=lambda backend: backend.signature_hash_supported(
+            hashes.SHA1()
```

Comment:
This can be sha256 if you use a different cert than `post2000utctime` for your no-SCT extension test. `x509/cryptography.io.pem` would work?